cmd/cli/main.go

```diff
@@ -110,7 +110,7 @@ func execute() error {
 	case 3:
 		instance = kubernetes.NewOIDCInstance(kubernetes.DefaultOIDCAttribute, kubernetes.DefaultAppKeyAttribute)
 	default:
-		return fmt.Errorf("unsupported authentiction type provided")
+		return fmt.Errorf("unsupported authentication type provided")
```

Comment:
Don't you use `git commit --fixup <thecommit>`? That already sets up the commit message to mark it as a fixup and on rebase it gets automatically linked to amend the right commit. (just saying because you missed the `i` and because a commit id rapidly loses its meaning after the odd rebase where you forgot to actually squash the commit).